src/commands/helper.py

```diff
@@ -22,6 +24,15 @@ def create_logging_embed(interaction: discord.Interaction, **kwargs):
                 case _:
                     fields.append(EmbedField(key.title(), value))
 
+    # Add "Messages Deleted" field if delete_messages_flag is provided
+    if delete_messages_flag is not None:
+        fields.append(
+            EmbedField(
+                "Messages Deleted",
+                "Yes" if delete_messages_flag else "No",
+            )
+        )
+
```

Comment:
i think this should be included in `kwargs`. `kwargs` is basically a generic catch all variable to deal with all the various cases such as the match/case statements above. we should also throw this up there as well using `kwargs`

Reply:
i cant really add a way to do this so the output will be a little eh.

We run into a pylance issue since there would be irrefutable cases and python only allows 1 in match/case.

![image](https://github.com/user-attachments/assets/0b2b69a5-213f-42ff-ac2a-9d20ce698292)

Reply:
you could make an object that holds some more information just like in the discord.Member case

Reply:
Or what you could do i sjust replace the title with actual text instead of using the key

Reply:
how would i replace the title with actual text

Reply:
``` 
                case _:  # Irrefutable pattern for all other types
                    if "delete_messages_flag" in key:
                        fields.append(EmbedField("Messages Deleted", value))
                    else:
                        fields.append(EmbedField(key.title(), value))
```